app/v1/dbControllers/QuestionController.js

```diff
@@ -96,5 +96,55 @@ class QuestionController {
 				});
 			});
 	}
+
+	/**
+	 *Upvote a question
+	 *
+	 * @static
+	 * @param {object} request
+	 * @param {object} response
+	 * returns object
+	 * @memberof QuestionController
+	 */
+	static upvoteQuestion(request, response) {
+		const { id } = request.params;
+
+		const selectQuery = { text: 'SELECT * FROM questions WHERE question_id = $1', values: [id] };
+
+		pool.query(selectQuery)
+			.then((result) => {
+				if (result.rows.length < 1) {
+					return response.status(404).json({
+						status: 404,
+						success: false,
```

Comment:
You don't need the success here, please adhere to the recommended response format.